CONTRIBUTING.md

```diff
@@ -68,3 +68,27 @@ The last line is the most important. `AssertFullSemver` will run GitVersion and
 
 ## 6. Submit a pull request with the failing test
 Even better include the fix, but a failing test is a great start
+
+
+# Build / Release Process
+We use Cake for our build and deployment process. The way the build / release process is setup is:
+
+1) We build releasable artifacts on AppVeyor
+1) Login to AppVeyor
+1) Deploy the latest master build
+![docs/img/release-1-deploy.png](docs/img/release-1-deploy.png)
+1) Choose GitVersion release, when you press deploy it will create a GitHub release (but it *will not* publish it)
+![docs/img/release-2-deploy.png](docs/img/release-2-deploy.png)
+1) All the artifacts should upload nicely
+![docs/img/release-3-deploy.png](docs/img/release-3-deploy.png)
+1) Head over to GitHub releases, you should have a draft release, download a copy of the release notes
+![docs/img/release-4-deploy.png](docs/img/release-4-deploy.png)
+1) Edit the release and do the following:
+    1. Remove the build metadata from the tag and title (the + and everything after it)
+    2. Paste the downloaded release notes in, you can clean them up if you want otherwise there may be closed issues which were questions etc. 
+    3. Tick the pre-release box if it's pre-release
+    4. Press Publish
+1) Publishing tags (a git tag) the release commit, this will trigger another appveyor build which only builds tags, this build uses deploy.cake. It downloads the artifacts from that GitHub release, then performs the release
```

Comment:
I don't really understand this point. Isn't the tag automatically created through publishing the GitHub release (point before)? Are there additional tags which needs to be created?

Reply:
Will clarify in the docs.

We push a *non published* github release, so it doesn't create a tag. The tag is created when you publish the release from the GitHub site

Reply:
Sorry, AppVeyor creates a draft, I wanted to call out that publish at this point creates the tag and that runs the build